src/components/ui4/CryptoIconUi4.tsx

```diff
@@ -115,6 +125,31 @@ const CryptoIconComponent = (props: Props) => {
         }
   }, [size, theme])
 
+  //
+  // Effects
+  //
+
+  React.useEffect(() => {
+    if (primaryCurrencyIconUrl == null || onIconColor == null) return
+
+    getColors(primaryCurrencyIconUrl, {
+      cache: true,
+      key: primaryCurrencyIconUrl
+    })
+      .then(colors => {
+        if (colors.platform === 'ios') {
+          console.log(colors)
+          onIconColor(colors.primary)
+        }
+        if (colors.platform === 'android') {
+          onIconColor(colors.dominant)
+        }
+      })
+      .catch(err => {
+        console.error(err)
```

Comment:
`console.error` gives the red screen of doom. Please use `console.log` or `console.warn` instead.